code/modules/power/singularity/field_generator.dm

```diff
@@ -72,9 +72,9 @@ no power level overlay is currently in the overlays list.
 
 
 /obj/machinery/field/generator/Initialize(mapload)
+	AddElement(/datum/element/blocks_explosives)
 	. = ..()
 	AddElement(/datum/element/empprotection, EMP_PROTECT_SELF | EMP_PROTECT_WIRES)
-	AddElement(/datum/element/blocks_explosives)
```

Comment:
I can't really test post init, which I would prefer to here. I don't really like that this is forced, even tho it really should not be

Reply:
I'm not really sure why it's like this. Windows adds the element before parent call so I'm just doing that to hopefully pass the tests.

Reply:
The element is required, and I need to test in init to ensure we never miss a block add.
I don't have a good way to like.... check after each atom's init tho, so I can't do what I want and let you just do what you wanted to. I'm sorry

Reply:
Not really hindering me, just looks kinda weird.